Source/Libraries/CorruptCore/CorruptCore.csproj

```diff
@@ -44,7 +44,6 @@
     <PlatformTarget>x64</PlatformTarget>
     <LangVersion>latest</LangVersion>
     <ErrorReport>prompt</ErrorReport>
-    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
```

Comment:
I need to check if I need to add the new ruleset to this file.

Reply:
It looks like the rulesets from elsewhere propagate to this `csproj` so I shouldn't need to make anymore changes here.